Regressions/Tests/network_electric/electric_btms_test.py

```diff
@@ -27,9 +27,10 @@
 # ----------------------------------------------------------------------------------------------------------------------  
 def main():           
          
-    battery_types = ['lithium_ion_lfp', 'lithium_ion_nmc']
+    battery_types = ['lithium_ion_nmc', 'lithium_ion_lfp']
     btms_types    = ['Liquid_Cooled_Wavy_Channel', 'Air_Cooled', None] 
-    CL_true       = [0.63136618] 
+    CL_true       = [[0.6044019730641954,0.604775958081217,0.604775958081217],
+                     [0.6044019723858562, .6047759580812461, 0.6047759580812461]] 
```

Comment:
change in cl values due to addition of cooling drag, deviation from before is minor (+0.01)